README.md

```diff
@@ -9,15 +9,12 @@ This project is released under the [MIT License](LICENSE).
 Contributing
 ============
 
-This project welcomes contributions and suggestions. Most contributions require you to
-agree to a Contributor License Agreement (CLA) declaring that you have the right to,
-and actually do, grant us the rights to use your contribution. For details, visit
-https://cla.microsoft.com.
+This project welcomes contributions and suggestions. All contributions to the Open Enclave SDK
+must adhere to the terms of the [Developer Certificate of Origin (DCO)](https://developercertificate.org/).
+For details, see parent project [Contributing to Open Enclave](https://github.com/openenclave/openenclave/blob/master/docs/Contributing.md).
 
 When you submit a pull request, a CLA-bot will automatically determine whether you need
 to provide a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the
 instructions provided by the bot. You will only need to do this once across all repositories using our CLA.

```

Comment:
This isn't really true anymore

Reply:
Hey @jhand2, thanks for taking the time.

Should CLA bot be changed to DCO in this case? or would you recommend removal?

Reply:
I think replacing it with the DCO github app (https://github.com/apps/dco) which is already done in the main repo is sufficient.

We should probably add it to our other repos (like this one and openenclave-site) too for consistency. Even though there aren't any external contributors today, there likely will be in the future.

Reply:
Done!

DCO app is present and the text has been modified.